locksmith/src/controllers/v2/stripeController.ts

```diff
@@ -36,3 +41,20 @@ export const connectStripe = async (req: Request, res: Response) => {
     })
   }
 }
+
+export const getConnectionsForManager = async (req: Request, res: Response) => {
+  const userAddress = req.user!.walletAddress
+  const connections = await stripeOperations.getConnectionsForManager(
+    userAddress
+  )
+  if (!connections) {
+    return []
+  }
+  return res.json(
```

Comment:
want to send something like `result`. We use this everywhere to be consistent so if we have any error but not complete failure, we could attach it as error or additional information later without changing the format.

Reply:
yes I can! Good point. Thanks!